src/media/class-wpml-elementor-media-node-provider.php

```diff
@@ -56,6 +56,10 @@ public function get( $type ) {
 					$node = new WPML_Elementor_Media_Node_WP_Widget_Media_Gallery( $this->media_translate );
 					break;
 
+				case 'all_widgets':
+					$node = new \WPML\PB\Elementor\Modules\AllWidgets( $this->media_translate );
```

Comment:
@GeorgeBotsev, maybe it would be more accurate to call this `AllNodes`. I am not sure about the terminology but I understand that this applies to all "nodes" and I am not sure that all "nodes" are "widgets".

Reply:
ok, I renamed the file and case.